packages/react/src/Tabs/Tabs.tsx

```diff
@@ -181,24 +183,9 @@ const _TabsRoot = forwardRef<HTMLDivElement, TabsProps>(
 
       // On narrow viewports this will scroll the active tab into the center.
       // Important because we show a gradient in dark mode.
-      const tabsContainer = tabsContainerRef.current
-      const containerWidth = tabsContainer.clientWidth
-      const containerPaddingRem = window
-        .getComputedStyle(tabsContainer)
-        .getPropertyValue('--brand-Tabs-container-padding')
-      const containerPadding = parseFloat(containerPaddingRem) * 16
-      const tabLeft = activeTabButton.offsetLeft - containerPadding
-      const tabWidth = activeTabButton.offsetWidth
-      const tabCenter = tabLeft + tabWidth / 2
-
-      const targetScrollLeft = tabCenter - containerWidth / 2
-
-      const maxScrollLeft = tabsContainer.scrollWidth - containerWidth // defines boudnary
-      const finalScrollLeft = Math.max(0, Math.min(targetScrollLeft, maxScrollLeft))
-
-      tabsContainer.scrollTo({
-        left: finalScrollLeft,
+      activeTabButton.scrollIntoView({
```

Comment:
I've tested this and the functionality appears to be the same, but another set of 👀 would be appreciated